public/app/features/alerting/NotificationsListPage.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, FC, useEffect } from 'react';
+import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
+import Page from 'app/core/components/Page/Page';
+import { getBackendSrv } from '@grafana/runtime';
+import { useAsyncFn } from 'react-use';
+import { useNavModel } from 'app/core/hooks/useNavModel';
+import { HorizontalGroup, Button, LinkButton } from '@grafana/ui';
+import { AlertNotification } from 'app/types/alerting';
+
+const deleteNotification = async (id: number) => {
+  return await getBackendSrv().delete(`/api/alert-notifications/${id}`);
+};
+
+const getNotifications = async () => {
+  return await getBackendSrv().get(`/api/alert-notifications`);
+};
+
+const NotificationsListPage: FC = () => {
+  const navModel = useNavModel('channels');
+
+  const [notifications, setNotifications] = useState<AlertNotification[]>();
```

Comment:
Maybe initialize this to an empty array so we don't have to deal with `undefined` state?
```suggestion
  const [notifications, setNotifications] = useState<AlertNotification[]>([]);
```